src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+// const TodoForm = props => <textarea placeholder="Enter stuff here"></textarea>
+
+// const AddToDo = props => <button>Add stuff</button>
+
+// const TodoForm = () => {
+//     return (
+//         <div>
+//             <form onSubmit={props.addToDo} placeholder="Enter stuff here"></form>
+//             <button>Add Todo</button>
+//             <button>Clear Completed</button>
+//         </div>
+//     )
+// }
+
+// const TodoForm = props => {
+//     return (
+//       <div>
+//         <form onSubmit={props.addCharacter}>
+//           <input
+//             type="text"
+//             name="inputText"
+//             placeholder="add new character"
+//             value={props.inputText}
+//             onChange={props.handleChange}
+//           />
+//           <input
+//             type="text"
+//             name="inputTextTwo"
+//             placeholder="Just here for looks"
+//             value={props.inputTextTwo}
+//             onChange={props.handleChange}
+//           />
+//           <button type="submit">Add Character</button>
+//         </form>
+//       </div>
+//     );
+//   };
+
+const CharacterForm = props => {
+    return (
+      <div>
```

Comment:
This would be a great place to use a `<Fragment>` since all your using this div for is a wrapper for containing your form and not using it for styling or anything.